README.md

```diff
@@ -1,3 +1,25 @@
-# myDiary
-myDairy is an online journal that allows users to make daily journal entries.
-The site is published at https://sammuel09.github.io/myDiary/
+# Project Title
+myDiary is a web-based journal application. 
```

Comment:
you should review the grammar here, a `web application` is more descriptive than  `web-based`.

Reply:
This is noted.